packages/app-content-pages/src/screens/Team/TeamContainer.js

```diff
@@ -32,8 +32,12 @@ TeamContainer.getInitialProps = async ({ req }) => {
   let error
   let teamData = []
   try {
+    console.log('FETCHING THE TEAM API DATA')
```

Comment:
These will go - leaving here for logging the known errors. Perhaps we should better handle the unavailable API proxy error and thus the error reporting on the home page.

Reply:
Does the original error get shown or is it replaced by a generic “something went wrong” message. Maybe show the specific error message on the page, after the generic error?

Reply:
The only error that's shown is the one translated one by https://github.com/zooniverse/front-end-monorepo/blob/60d70e1c83bd991461e134a21499a50aa6895ab8/packages/app-content-pages/src/screens/Team/TeamContainer.js#L21

Reply:
I'd add the error message from the request error, if you want a more useful message displayed in the browser.